pkg/pillar/cmd/ledmanager/ledmanager.go

```diff
@@ -452,9 +451,8 @@ func ExecuteDDCmd(ledName string) {
 }
 
 func doDD(count int) {
-	cmd := exec.Command("dd", "if="+diskDevice, "of=/dev/null", "bs=4M",
-		fmt.Sprintf("count=%d", count), "iflag=nocache")
-	stdout, err := cmd.Output()
+	stdout, err := base.ExecCommand(log, false, false,
+		"dd", "if="+diskDevice, "of=/dev/null", "bs=4M", fmt.Sprintf("count=%d", count), "iflag=nocache")
```

Comment:
Hey @eriknordmark -- remind me if there was any reason to use dd here as opposed to simply writing to that file from native go io library?

Reply:
@rvs We don't want to write since that causes wear. Thus we read. But normally reads are satisfied by the cache, Hence the nocache flag. I guess we could figure out what dd does with the noocache flag and avoid dd.

Reply:
>  I guess we could figure out what dd does with the noocache flag and avoid dd.

That's exactly my point @eriknordmark -- and here's what it does ;-)

```
fadvise64(0, 0, size of buffer, POSIX_FADV_DONTNEED) = 0
```

So a combination of:

```
unix.Fadvise(..., 4) // POSIX_FADV_DONTNEED
syscall.Madvise(..., 4) // MADV_DONTNEED
```

Should be able to replace this nasty dd invocation.

@adarsh-zededa -- are you up to doing this extra change?